src/main/java/org/stroganov/ws/LibraryServerServiceImpl.java

```diff
@@ -0,0 +1,141 @@
+package org.stroganov.ws;
+
+import jakarta.jws.HandlerChain;
+import jakarta.jws.WebParam;
+import jakarta.jws.WebService;
+import org.stroganov.dao.LibraryDAO;
+import org.stroganov.entities.*;
+import org.stroganov.util.DataManager;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+@HandlerChain(file = "../handler.xml")
+@WebService(endpointInterface = "org.stroganov.ws.LibraryService")
+public class LibraryServerServiceImpl implements LibraryService {
+
+    private final LibraryDAO libraryDAO = DataManager.getLibraryDAO();
+
+    @Override
+    public User[] getNewUser(@WebParam(name = "userName") String name) {
+        ArrayList<User> stringArrayListUsers = new ArrayList<>();
```

Comment:
```suggestion
        List<User> stringArrayListUsers = new ArrayList<>();
```

Reply:
It was method only for test. I deleted it.